plugins/modules/dcnm_vrf.py

```diff
@@ -1056,7 +1075,14 @@ def get_diff_merge(self):
                         if not vrf_id_obj['DATA']:
                             continue
 
-                        vrf_id = vrf_id_obj['DATA'].get('partitionSegmentId')
+                        if self.dcnm_version == 11:
+                            vrf_id = vrf_id_obj['DATA'].get('partitionSegmentId')
+                        elif self.dcnm_version >= 12:
+                            vrf_id = vrf_id_obj['DATA'].get('l3vni')
+                        else:
+                            msg = "Unsupported DCNM version"
+                            self.module.fail_json(msg)
```

Comment:
Please include version number in the error message.

Reply:
Added the version number to error message